frontend/src/app/components/ExperimentsListPage.js

```diff
@@ -46,9 +53,14 @@ export default class ExperimentsListPage extends React.Component {
 
         <div className="pinstripe responsive-content-wrapper"></div>
         <div className="responsive-content-wrapper">
-          <div data-hook="experiment-list">
-            <ExperimentCardList {...this.props} eventCategory="HomePage Interactions" />
-          </div>
+          <ExperimentCardList {...this.props} experiments={currentExperiments} eventCategory="HomePage Interactions" />
+          {pastExperiments.length > 0 &&
+          <div className={classnames(['button', 'outline'])}
+              style={{margin: '0 auto', display: 'table'}}
+              onClick={() => this.setState({ showPastExperiments: true })}
+              data-l10n-id="viewPastExperiments">View Past Experiments</div>}
```

Comment:
I need help on how to style this button to get it in the right spot. I had success adding the `style` above but before I put it in scss I'd like to know if that's a *good* way to do it. @johngruen @chuckharmston ?